src/headers.ts

```diff
@@ -14,10 +14,59 @@
  * limitations under the License.
  */
 
+import { AcrolinxEndpointProps } from 'src';
+import { AccessToken, ServiceType, StringMap } from './common-types';
+
 export const HEADER_X_ACROLINX_CLIENT = 'X-Acrolinx-Client';
 export const HEADER_X_ACROLINX_AUTH = 'X-Acrolinx-Auth';
 export const HEADER_X_ACROLINX_BASE_URL = 'X-Acrolinx-Base-Url';
 export const HEADER_X_ACROLINX_CLIENT_LOCALE = 'X-Acrolinx-Client-Locale';
 export const HEADER_X_ACROLINX_APP_SIGNATURE = 'X-Acrolinx-App';
 
 export const HEADER_ACROLINX_ONE_AUTH = 'Authorization';
+
+export function getCommonHeaders(
+  props: AcrolinxEndpointProps,
+  accessToken?: AccessToken,
+  serviceType?: ServiceType,
+): StringMap {
+  const headers: StringMap = {
+    'Content-Type': 'application/json',
+  };
+  if (serviceType === ServiceType.ACROLINX_ONE) {
```

Comment:
make this default and wrap ServiceType.ACROLINX_CORE part in condition so in future removal will be easy.